Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/BaseClasses/Effectiveness.mo

```diff
@@ -0,0 +1,163 @@
+within Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses;
+model Effectiveness
+  "Model for calculating the heat exchange effectiveness"
+  extends Modelica.Blocks.Icons.Block;
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(final max=1)
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(final max=1)
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(final max=1)
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode";
```

Comment:
It is nowhere explained what the 75% relate too. Mass flow rate, or rotational speed?